src/tools/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ add_executable(ceph_radosacl radosacl.cc)
 target_link_libraries(ceph_radosacl librados global)
 install(TARGETS ceph_radosacl DESTINATION bin)
 
+install(PROGRAMS
```

Comment:
could you update the commit message with the rationale you put in the PR's comment?

Reply:
@tchaikov updated as you suggested, thx!